onyo/lib/inventory.py

```diff
@@ -124,15 +124,15 @@ def __init__(self, repo: OnyoRepo) -> None:
 
     @property
     def root(self):
-        """Path to root inventory directory"""
+        r"""Path to root inventory directory"""
         return self.repo.git.root
 
     def reset(self) -> None:
-        """throw away pending operations"""
+        r"""throw away pending operations"""
```

Comment:
Capitalization of "throw".

Reply:
Thanks for the catch. I have changed this to "Discard".

I have fixed it locally, but will not include it in this PR.